Translator.php

```diff
@@ -35,9 +35,10 @@
  */
 class Translator extends BaseObject
 {
-    private $_where;
-    private $_params = [];
-    private $_operators;
+    protected $_where;
```

Comment:
Why protected?

Reply:
need to extends from Translator

Reply:
you have `where()` and `params` method i can't see it's necessary

Reply:
when i  extends from Translator 
where() returns empty  because from child class has no access to private attribute

Reply:
Private variables, are variables that are visible only to the class to which they belong. Protected variables, are variables that are visible only to the class to which they belong, and any subclasses.

to extends from Translator  I need overwrite all   Translator

Reply:
just change    private  to protected

Reply:
and i correctly extends from your   Translator.
because right now i  overwrite all   Translator's methods
but need only buildWhere and add my